nautobot/core/graphql/types.py

```diff
@@ -1,9 +1,19 @@
 from django.contrib.contenttypes.models import ContentType
-
+import graphene
 import graphene_django_optimizer as gql_optimizer
 
 
-class ContentTypeType(gql_optimizer.OptimizedDjangoObjectType):
+class OptimizedNautobotObjectType(gql_optimizer.OptimizedDjangoObjectType):
```

Comment:
Probably need some documentation in `nautobot/docs/plugins/development.md` in the "Creating your own graphql type object" section?